{{cookiecutter.github_repository}}/mkdocs.yml

```diff
@@ -5,19 +5,21 @@ site_author: {{ cookiecutter.github_username }}
 repo_url: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.github_repository }}
 
 nav:
-- Home: 'index.md'
+- Home: index.md
 - API:
-  - 'Overview & Usages': 'api/overview.md'
-  - 'REST Endpoints': 'api/endpoints.md'
-  - 'Errors': 'api/errors.md'
-  - 'Changelog': 'api/changelog.md'
+  - 'Overview': api/0-overview.md
```

Comment:
The initial number is just convention or is it for a specific purpose?

Reply:
the numbers are more for sorting the files in the sidebar and keeping track of the sequence in the navigation in the doc. I felt it's better that way.

Reply:
Sounds good! Thanks!